master/internal/db/postgres_experiments.go

```diff
@@ -1010,16 +1012,18 @@ func ExperimentsTrialAndTaskIDs(ctx context.Context, idb bun.IDB, expIDs []int)
 }
 
 // ExperimentNumSteps returns the total number of steps for all trials of the experiment.
-func (db *PgDB) ExperimentNumSteps(id int) (int64, error) {
-	var numSteps int64
-	if err := db.sql.Get(&numSteps, `
-SELECT count(*)
-FROM raw_steps s, trials t
-WHERE t.experiment_id = $1 AND s.trial_id = t.id
-`, id); err != nil {
-		return 0, errors.Wrapf(err, "querying for number of steps of experiment %v", id)
+func ExperimentNumSteps(ctx context.Context, id int) (int64, error) {
+	numSteps, err := Bun().NewSelect().
+		TableExpr("raw_steps AS s").
+		Join("JOIN trials AS t ON t.id = s.trial_id").
+		Where("t.experiment_id = ?", id).
+		Count(ctx)
+
```

Comment:
```suggestion
```

I would remove this newline just so that the error handling is right next to the code that produces the error